src/styles/main.scss

```
@@ -1,3 +1,290 @@
+  grid-row-start: 1;
+  grid-row-end: 2;
+}
+
+.nav__list {
+  padding: 0;
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  list-style-type: none;
+  margin: 0;
+}
+
+.nav__logoBox {
+  display: block;
+  height: 40px;
+}
+
+.nav__logo {
+  width: 40px;
+  height: 40px;
+}
+
+.nav__item {
+  position: relative;
+  list-style: none;
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin-right: 20px;
+  line-height: 15px;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  color: #000;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
+  line-height: 15px;
+}
+
+.is-active {
+  position: relative;
+  color: var(--blue);
+}
+
+.nav__link:hover {
+  color: var(--blue);
+  transition: all 300ms;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 4px;
+  background-color: var(--blue);
+  border-radius: 8px;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid var(--lightgrey);
+  background: var(--white);
+}
+
+.card :hover {
+  color: #34568b;
+}
+
+.card--1 {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.card--2 {
+  grid-column-start: 4;
+  grid-column-end: 5;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.card--3 {
+  grid-column-start: 6;
+  grid-column-end: 7;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.card--4 {
+  grid-column-start: 8;
+  grid-column-end: 9;
+  grid-row-start: 2;
+  grid-row-end: 3;
+}
+
+.card--5 {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 4;
+  grid-row-end: 5;
+}
+
+.card--6 {
+  grid-column-start: 4;
+  grid-column-end: 5;
+  grid-row-start: 4;
+  grid-row-end: 5;
+}
+
+.card--7 {
+  grid-column-start: 6;
+  grid-column-end: 7;
+  grid-row-start: 4;
+  grid-row-end: 5;
+}
+
+.card--8 {
+  grid-column-start: 8;
+  grid-column-end: 9;
+  grid-row-start: 4;
+  grid-row-end: 5;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  flex-shrink: 0;
+  margin: 0 auto;
+}
+
+.card__description {
+  color: var(--darkgrey);
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin-top: 40px;
+  margin-bottom: 0;
+}
+
+.card__code {
+  color: var(--grey);
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin-top: 4px;
+  margin-bottom: 0;
+  align-self: start;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  align-self: stretch;
+}
+
+.card__price-text {
+  display: inline-block;
+  color: var(--grey);
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0;
+}
+
+.card__price-value {
+  display: inline-block;
+  color: var(--darkgrey);
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  margin: 0;
+}
```

Need to fix it everywhere
```suggestion
.card__price {
  display: flex;
  justify-content: space-between;
  margin-top: 24px;
  align-self: stretch;
  
  &-text {}
  
  &-value {}
}
```